programs/fuzz/fuzz_pubkey.c

```diff
@@ -20,7 +20,9 @@ int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
             mbedtls_mpi_init( &DQ ); mbedtls_mpi_init( &QP );
 
             rsa = mbedtls_pk_rsa( pk );
-            ret = mbedtls_rsa_export( rsa, &N, &P, &Q, &D, &E );
+            if ( mbedtls_rsa_export( rsa, &N, &P, &Q, &D, &E ) != 0 ) {
+                abort();
+            }
             ret = mbedtls_rsa_export_crt( rsa, &DP, &DQ, &QP );
```

Comment:
And please check `ret` for `mbedtls_rsa_export_crt` here, but expecting `MBEDTLS_ERR_RSA_BAD_INPUT_DATA`.

Reply:
Like this ?

Reply:
Yes. Looks good to me, thanks.